semgrep-core/src/naming/Naming_AST.ml

```diff
@@ -506,6 +506,89 @@ let resolve lang prog =
                   (* TODO? Maybe we need a `with_new_class_scope`. For now, abusing `with_new_function_scope`. *)
                   with_new_function_scope (special_class_params @ class_params)
                     env.names (fun () -> k x))
+          (* const x = require('y');
```

Comment:
I would clarify that `require('y')` in Javascript behaves as a local import here